pkg/synopsysctl/cmd_create.go

```diff
@@ -315,15 +318,24 @@ var createBlackDuckCmd = &cobra.Command{
 		}
 
 		log.Infof("creating Black Duck '%s' in namespace '%s'...", blackDuckName, blackDuckNamespace)
+		version := blackDuck.Spec.Version
+		if len(version) == 0 {
+			versions := apps.NewApp(&protoform.Config{}, restconfig).Blackduck().Versions()
```

Comment:
Why not letting the operator select the latest version instead of doing so in synopsysctl?

Reply:
This validation need to be done in synopsyctl, otherwise the CR will be created. That's why I need to get the version to determine it...